web-ui/src/components/guild-results/GuildsActions.jsx

```diff
@@ -1,60 +0,0 @@
-import React, { useContext, useState } from 'react';
```

Comment:
This is a very small component that is only used in `GridResults.jsx` which needed feedback from this.
It seemed easiest to just move the functionality into `GridResults.jsx` than to add more props to communicate.